.jenkins/pytorch/build.sh

```diff
@@ -30,13 +30,26 @@ cmake --version
 pip install -r requirements.txt || true
 
 if [[ "$BUILD_ENVIRONMENT" == *rocm* ]]; then
+  # This is necessary in order to cross compile (or else we'll have missing GPU device).
   export HCC_AMDGPU_TARGET=gfx900
+
+  # These environment variables are not set on CI when we were running as the Jenkins user.
+  # The HIP Utility scripts require these environment variables to be set in order to run without error.
   export LANG=C.UTF-8
   export LC_ALL=C.UTF-8
+  export PYTORCH_BUILT_WITH_ROCM=1
+
+  # This environment variable enabled HCC Optimizations that speed up the linking stage.
+  # https://github.com/RadeonOpenCompute/hcc#hcc-with-thinlto-linking
+  # export KMTHINLTO=1
```

Comment:
Why is this commented out? When should it be uncommented?

Reply:
OK. Always good to put that in a comment, in case the commented out lines are more durable than you expect ;)